### PR TITLE
Doc index

### PR DESCRIPTION
**Pull request by babsey**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -60,62 +60,80 @@ Please keep this in mind when searching for information on a specific version of
...
 
-:General: |docs| |license| |doi|
-:GitHub: |github-commit-activitiy| |github-stars| |github-forks|
-:DockerHub: |dockerhub-version| |dockerhub-image-size| |dockerhub-pulls|
-:PyPI: |pypi-version| |pypi-dm|
+:General: |docs| |license| |paper|
+:GitHub: |github-release| |github-commit-activitiy| |github-forks| |github-stars|
+:Docker: |docker-version| |docker-image-size| |docker-pulls|
+:Python: |python-version| |python-downloads|
 :Anaconda: |anaconda-version| |anaconda-downloads|
```
Comment (jens-br) [inline docs/index.rst:67]: You changed it to ``Conda` below. Did you mean this?

```suggestion
:Conda: |conda-version| |conda-downloads|
```

File: docs/index.rst [hunk 2]
```diff
@@ -60,62 +60,80 @@ Please keep this in mind when searching for information on a specific version of
...
 
-.. |pypi-dm| image:: https://img.shields.io/pypi/dm/nest-desktop.svg
-   :alt: Downloads per month
+.. |python-downloads| image:: https://img.shields.io/pypi/dm/nest-desktop.svg
+   :alt: Python downloads per month
    :target: https://pypi.org/project/nest-desktop/
 
-.. |anaconda-version| image:: https://anaconda.org/conda-forge/nest-desktop/badges/version.svg
-   :alt: Latest version on Anaconda
+.. |conda-version| image:: https://img.shields.io/conda/v/conda-forge/nest-desktop.svg&logo=conda-forge&logoColor=white
```
Comment (jens-br) [inline docs/index.rst:119]: It does not work and - if included -  shows only this:
grafik

Did you mean this?
```suggestion
.. |conda-version| image:: https://anaconda.org/conda-forge/nest-desktop/badges/version.svg
```

File: docs/index.rst [hunk 3]
```diff
@@ -60,62 +60,80 @@ Please keep this in mind when searching for information on a specific version of
...
+   :alt: Conda version
    :target: https://anaconda.org/conda-forge/nest-desktop
```
Comment (jens-br) [inline docs/index.rst:121]: For the version entry, we could direct them to the list of all versions. This is not mandatory, only a suggestion to think about. (What would you prefer? I am not sure.)
```suggestion
   :target: https://anaconda.org/conda-forge/nest-desktop/files
```

**Review by jens-br — CHANGES_REQUESTED** [review]
Good additions, I would suggest only minor changes.

File: docs/index.rst (hunk 3 above)
Comment (babsey) [reply docs/index.rst:121]: A main page is sufficient. It also gives an overview.

**Review by jens-br — APPROVED** [review]
LGTM :+1:

File: docs/index.rst (hunk 3 above)
Comment (jens-br) [reply docs/index.rst:121]: Ok :+1:

**Pull request merged**